### PR TITLE
Rating Component

### PR DESCRIPTION
**Pull request by bajajcodes**
- Star rating component, on selection colour changes to dark orange and on revert back to black colour.
- For star Unicode character code is used.
- For changing colours JavaScript and DOM API is used.

https://user-images.githubusercontent.com/29247011/162191466-0a005b3a-06ea-4a2e-86b9-678757dfa2fd.mp4

### CODE REVIEW

File: pages/heading.html [hunk 1]
```diff
@@ -1,117 +1,158 @@
...
-    <link rel="icon" href="../assets/logo/logo.svg" type="image/svg+xml">
-    <link rel="stylesheet" href="../css/devui.css">
-    <link rel="stylesheet" href="../css/docs.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" href="../assets/logo/logo.svg" type="image/svg+xml" />
+    <link rel="stylesheet" href="../css/devui.css" />
+    <link rel="stylesheet" href="../css/docs.css" />
     <script src="../scripts/navbar.js" defer></script>
```
Comment (pradipgharge) [inline pages/heading.html]: You can add the script tags at the bottom of Html file

File: css/components/modal.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+.modal_body {
+  min-height: calc(var(--size-9) * 3);
+  max-height: calc(var(--size-9) * 6);
+  overflow-x: hidden;
+  overflow-y: auto;
+  background: var(--white);
+}
+
+.modal_footer {
+  display: flex;
```
Comment (saifudinbadani) [inline css/components/modal.css:53]: if you are using flex property repeatedly, I suggest you make a class of it.

File: scripts/rating.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    });
+  });
+});
+
+function SetRatingStar(rating, stars) {
+  let len = stars.length;
+
+  for (var i = 0; i < len; i++) {
+    if (i < rating) {
+      stars[i].style.color = "darkorange";
```
Comment (saifudinbadani) [inline scripts/rating.js:18]: this code works fine, but once you have clicked , you cannot deselect them all at once.

**Review by saifudinbadani — APPROVED** [review]
Rest looks fine to me.

File: scripts/rating.js (hunk 3 above)
Comment (bajajcodes) [reply scripts/rating.js:18]: This I didn't thought, I will implement it.

**Review by krate27 — APPROVED** [review]
Code Looks good to merge. Really liked how you have used variables and formatted code beautifully. Great folder structure too. 
Good to go from my side.

**Review by pradipgharge — APPROVED** [review]
suggested some changes, rest looks good to merge

**Pull request merged**